app/Http/Controllers/User/ReviewBookController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use App\Http\Controllers\Controller;
+use App\Repositories\Contracts\ReviewBookRepository;
+use App\Http\Requests\ReviewRequest;
+use Illuminate\Http\Request;
+use App\Repositories\Contracts\BookRepository;
+use Auth;
+use Exception;
+
+class ReviewBookController extends Controller
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ReviewBookController.php</br>Opening brace should be on a new line